src/components/TabsPage.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tab: Tab[],
+};
+
+export const TabsPage: FC<Props> = ({ tab }) => {
+  const { tabId } = useParams();
+  const selectedTabId = tab.find(item => tabId === item.id)?.content;
```

Comment:
This line is finding the tab content based on the id from the URL. But it's named as 'selectedTabId', which is misleading. It should be named 'selectedTabContent' as it represents the content of the selected tab and not the id.